pgbot/__init__.py

```diff
@@ -299,9 +299,14 @@ async def handle_message(msg: discord.Message):
             await embed_utils.send(
                 common.entries_discussion_channel, title, "", color, fields=fields
             )
+        elif (
+            random.random() < await emotion.get("depression") / 200
+            or (msg.author.id == 683852333293109269 and not common.TEST_MODE)
```

Comment:
You don't need to check for `not common.TEST_MODE` again here and in the block below this function, because this whole block does not run on TESTMODE anyways

Reply:
ah yeah, I forgot about that since I commented out the `not common.TEST_MODE` so much :kekw: